packages/abstract-interface/src/native/ans_host.rs

```diff
@@ -64,9 +63,9 @@ impl AnsHost<Daemon> {
         let file =
             File::open(&path).unwrap_or_else(|_| panic!("file should be present at {}", &path));
         let json: serde_json::Value = from_reader(file)?;
-        let chain_id = self.get_chain().state().chain_id.clone();
+        let chain_id = self.get_chain().state().chain_data.chain_name;
```

Comment:
New cw-orchestrator chain info interface

Reply:
I think the two got mixed up here, chain_id is assigned to network_id and reverse

Reply:
chain_name is network_id